### PR TITLE
[release-0.65] cluster: Enable just ovs repo (#1163)

### PR DESCRIPTION
**Pull request by qinqon**
**What this PR does / why we need it**:
Some openstack repos does not support centos 8 stream yet. This change
point directly to the centos 8 stream ovs repo.

**Special notes for your reviewer**:
(cherry picked from commit 152a52e78f453e766e42fb590207ec4eb90cb3fe)

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Review by rhrazdil — APPROVED** [review]

File: cluster/up.sh [hunk 1]
```diff
@@ -35,12 +35,10 @@ fi
 if [[ "$KUBEVIRT_PROVIDER" =~ k8s- ]]; then
     echo 'Installing Open vSwitch on nodes'
     for node in $(./cluster/kubectl.sh get nodes --no-headers | awk '{print $1}'); do
-        ./cluster/cli.sh ssh ${node} -- sudo yum config-manager --set-enabled powertools
-        ./cluster/cli.sh ssh ${node} -- sudo yum install -y epel-release centos-release-openstack-wallaby epel-release centos-release-openstack-train https://rdoproject.org/repos/rdo-release.rpm
-        ./cluster/cli.sh ssh ${node} -- sudo yum install -y openvswitch libibverbs NetworkManager-ovs
+        ./cluster/cli.sh ssh ${node} -- sudo dnf install -y centos-release-nfv-openvswitch
+        ./cluster/cli.sh ssh ${node} -- sudo dnf install -y openvswitch2.16 libibverbs
```
Comment (rhrazdil) [inline cluster/up.sh:39]: Why are the NM packages ommited?

Comment (qinqon) [reply cluster/up.sh:39]: Fixing

**Comment by qinqon** [comment]
Pinned to NM 1.34 like knmstate

**Review by rhrazdil — APPROVED** [review]

**Pull request merged**